build.gradle

```diff
@@ -232,7 +232,10 @@ subprojects {
 
   apply plugin: 'java-library'
   apply plugin: 'checkstyle'
-  apply plugin: "com.github.spotbugs"
+
+  // spotbugs doesn't support Java 21 yet
+  if (!JavaVersion.current().isCompatibleWith(JavaVersion.VERSION_21))
```

Comment:
Could we please add the following here:

`println 'Disabling spotbugs plugin since it is not compatible with current version of JVM'`

It will help identify when spotbugs is being actively disabled.

Reply:
I see this as a short term thing, so didn't think it was worth it. I expect spotbugs to have a release that works with Java 21 shortly. If it doesn't happen in the next few weeks, happy to see the warning. Is that a reasonable compromise.

Reply:
Sure. As I mentioned, I am fine with not adding this as well.